build-settings/webpack.config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-commonjs */
```

Comment:
Thank you for adding these!

Reply:
I think eventually we'll want want to have two different eslint configs, one for node and one for browser.  We should be able to configure eslint to use different configs based on a path regex.